common/components/chrome/SubHeader.jsx

```diff
@@ -262,23 +262,31 @@ class SubHeader extends React.Component<{||}, State > {
   }
 
   _renderHeaderLinks(): React$Node {
-    return FooterLinks.list().map((link, i) =>
-      <div className="SectionLink-root" key={i}>
-        <a className="SubHeader-anchor" href={link.url} target="_blank" rel="noopener noreferrer">
-          <h3>{link.name}</h3>
-        </a>
-      </div>
+    //TODO: Extract the links into a component much like FooterLinks
+    return (
+      <React.Fragment>
+        <div className="SectionLink-root">
+          <a className="SubHeader-anchor" href="https://connect.democracylab.org/" rel="noopener noreferrer">
+            <h3>About</h3>
+          </a>
+        </div>
+        <div className="SectionLink-root">
+          <a className="SubHeader-anchor" href="mailto:hello@democracylab.org" rel="noopener noreferrer">
+          <h3>Contact Us</h3>
+          </a>
+        </div>
+      </React.Fragment>
     );
   }
 
   _renderIcon(): React$Node {
     return (
-      this.state.user && this.state.user.user_thumbnail ?
-        <img className="SubHeader-userImg" src={this.state.user.user_thumbnail} /> :
+      this.state.user && !_.isEmpty(this.state.user.user_thumbnail) ?
```

Comment:
I ran into an issue with a logged-in user with no profile photo; just checking on `this.state.user.user_thumbnail` wasn't robust enough since an empty object returned true and then it expected an img src that didn't exist. So I used lodash's isEmpty for a slightly more robust conditional.